Ginger/GingerCore/Actions/UIAutomation/UIAComWrapperHelper.cs

```diff
@@ -476,6 +481,10 @@ public override List<AppWindow> GetListOfDriverAppWindows()
 
                     foreach (AutomationElement window in AppWindows)
                     {
+                        if (CheckUserSpecificProcess(window) == false)
```

Comment:
Add this check after IsWindowValid check. Becuase sometimes windows is not valid and trying to retrieve property will throw exceptin

Reply:
Pushed above changes